beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/aws/S3ClientTest.java

```diff
@@ -207,6 +207,21 @@ void deleteObjectsInDirectory() {
     assertThat(amazonS3.doesObjectExist(bucket, key2)).isFalse();
   }
 
+  @Test
+  void deleteThousandObjectsInDirectory() {
+    ArrayList<String> keys = new ArrayList<>();
+    int totalObjects = 1234;
+    for(int i = 1; i <= totalObjects; i++){
+      var key = key1+i;
+      amazonS3.putObject(bucket, key, content);
+      keys.add(key);
+    }
+
+    List<String> result = s3Client.deleteObjects(bucket, keys);
+
+    assertThat(result.size()).isEqualTo(totalObjects);
```

Comment:
you can probably assert the actual number of objects left by doing a get objects s3 call

Reply:
added the assertion